inventree_kicad/models.py

```diff
@@ -51,6 +51,35 @@ class Meta:
         help_text=_('Default value parameter template for this category, if not specified for an individual part'),
     )
 
+    footprint_parameter_template = models.ForeignKey(
```

Comment:
That's just the existing KICAD_FOOTPRINT_PARAMETER setting, but on a per KiCad category level.

Reply:
I'm uncertain about this particular case. We already have the settings parameter in place for that purpose. I can't envision a scenario where the footprint would be stored across multiple parameters instead of just one. It seems like that could become confusing quite rapidly.

Reply:
Well pretty much exactly for the reason this whole PR exists.

In my case for example, I've got a whole bunch of components in InvenTree already. Some of the (resistors, capacitors, etc.) already have a "Package Type" parameter set, which I can use with the introduced functionality here, to map from said "Package Type" to a KiCad Footprint. But that "Package Type" parameter isn't set for all parts. So for some more specialized parts, like Potentiometers or certain ICs or whatever, I'd prefer to actually just add a "KiCad Footprint" parameter directly, which I can change from the InvenTree UI.

So what I'd do is set the global `KICAD_FOOTPRINT_PARAMETER` setting to the "KiCad Footprint" parameter and then override it for certain categories (resistors and capacitors for example) to the "Package Type" parameter.

Reply:
Nothing confusing about that really, it's just a simple override.

Reply:
Done.